src/client/pythonEnvironments/base/info/env.ts

```diff
@@ -202,6 +202,17 @@ function getPythonVersionInfoHeuristic(version: PythonVersion): number {
     return infoLevel;
 }
 
+/**
+ * Compares two python versions, based on the amount of data each object has. If versionA has
+ * less information then the returned value is negative. If it is same then 0. If versionA has
+ * more information then positive.
+ * @param versionA
+ * @param versionB
+ */
+export function comparePythonVersionsByHeuristic(versionA: PythonVersion, versionB: PythonVersion): number {
+    return getPythonVersionInfoHeuristic(versionA) - getPythonVersionInfoHeuristic(versionB);
```

Comment:
```suggestion
    return Math.sign(getPythonVersionInfoHeuristic(versionA) - getPythonVersionInfoHeuristic(versionB));
```
We're only interested in whether it's positive, negative, or zero.